src/main/scala/firrtl/annotations/AnnotationUtils.scala

```diff
@@ -3,9 +3,15 @@
 package firrtl
 package annotations
 
+import net.jcazevedo.moultingyaml._
+import firrtl.annotations.AnnotationYamlProtocol._
+
 import firrtl.ir._
 
 object AnnotationUtils {
+  def toYaml(a: Annotation): String = a.toYaml.prettyPrint
+  def fromYaml(s: String): Annotation = s.parseYaml.convertTo[Annotation]
```

Comment:
FYI -- I'm using yaml to serialize/deserialize the annotation value string now too, so I can stick a lot of parameters into an Annotation. (I also think it's wonky that an Annotation *has* to be tied to a Transform).